### PR TITLE
vSphere with tanzu RA 2.1.0

### PR DESCRIPTION
**Pull request by rahulk10-vmware**
Please review and provide feedback.

### CODE REVIEW

File: src/reference-designs/tkg-nsxt-airgap-ra.md [hunk 1]
```diff
@@ -256,6 +256,40 @@ The following table provides a list of firewall rules based on the assumption th
...
+|TKO-ALB-008|Configure anti-affinity rules for the NSX ALB controller cluster.|This is to ensure that no two controllers end up in same ESXi host and thus avoid single point of failure.|Anti-Affinity rules need to be created manually.|
+|TKO-ALB-009|Configure backup for the NSX ALB Controller cluster.|Backups are required if the NSX ALB Controller becomes inoperable or if the environment needs to be restored from a previous state.|To store backups, a SCP capable backup location is needed. SCP is the only supported protocol currently.|
+|TKO-ALB-010|Create an NSX-T Cloud connector on NSX ALB controller for each NSX transport zone requiring load balancing.|An NSX-T Cloud connector configured on the NSX ALB controller provides load balancing for workloads belonging to a transport zone on NSX-T.|None.|
+|TKO-ALB-011|Configure Remote logging for NSX ALB Controller to send events on Syslog.|For operations teams to be able to centrally monitor NSX ALB and escalate alerts events must be sent from the NSX ALB Controller|  Additional Operational Overhead.<br> Additional infrastructure Resource.
```
Comment (reshmarajkumar-vmw) [inline src/reference-designs/tkg-nsxt-airgap-ra.md:277]:
```suggestion
|TKO-ALB-011|Configure Remote logging for NSX ALB Controller to send events on Syslog.|For operations teams to be able to centrally monitor NSX ALB and escalate alerts events must be sent from the NSX ALB Controller|  Additional Operational Overhead.<br> Additional infrastructure Resource. | 
```

File: src/reference-designs/tkg-nsxt-airgap-ra.md [hunk 2]
```diff
@@ -256,6 +256,40 @@ The following table provides a list of firewall rules based on the assumption th
...
+|TKO-ALB-009|Configure backup for the NSX ALB Controller cluster.|Backups are required if the NSX ALB Controller becomes inoperable or if the environment needs to be restored from a previous state.|To store backups, a SCP capable backup location is needed. SCP is the only supported protocol currently.|
+|TKO-ALB-010|Create an NSX-T Cloud connector on NSX ALB controller for each NSX transport zone requiring load balancing.|An NSX-T Cloud connector configured on the NSX ALB controller provides load balancing for workloads belonging to a transport zone on NSX-T.|None.|
+|TKO-ALB-011|Configure Remote logging for NSX ALB Controller to send events on Syslog.|For operations teams to be able to centrally monitor NSX ALB and escalate alerts events must be sent from the NSX ALB Controller|  Additional Operational Overhead.<br> Additional infrastructure Resource.  
+|TKO-ALB-012|Use LDAP/SAML based Authentication for NSX ALB| Helps to Maintain Role based Access Control | Additional Configuration is required
```
Comment (reshmarajkumar-vmw) [inline src/reference-designs/tkg-nsxt-airgap-ra.md:278]:
```suggestion
|TKO-ALB-012|Use LDAP/SAML based Authentication for NSX ALB| Helps to Maintain Role based Access Control | Additional Configuration is required|
```

File: src/reference-designs/tkg-nsxt-airgap-ra.md [hunk 3]
```diff
@@ -256,6 +256,40 @@ The following table provides a list of firewall rules based on the assumption th
...
+|TKO-ALB-SE-004|Enable 'Dedicated dispatcher CPU' on SE groups that contain the SE VMs of 4 or more vCPUs.<br>Note: This setting must be enabled on SE groups that are servicing applications that have high network requirement.|This enables a dedicated core for packet processing enabling high packet pipeline on the SE VMs.|None.|
+|TKO-ALB-SE-005|Dedicated Service Engine Group for the TKG Management|SE resources are guaranteed for TKG Management Stack and provides data path segregation for Management and Tenant Application |Dedicated service engine Groups increase licensing cost. </p>||
+|TKO-ALB-SE-006|Dedicated Service Engine Group for the TKG Workload Clusters Depending on the nature and type of workloads (dev/prod/test)|SE resources are guaranteed for single or set of workload clusters and provides data path segregation for Tenant Application hosted on workload clusters |Dedicated service engine Groups increase licensing cost. </p>|
+|TKO-ALB-SE-007|Set 'Placement across the Service Engines' setting to 'distributed'.|This allows for maximum fault tolerance and even utilization of capacity.| None
```
Comment (reshmarajkumar-vmw) [inline src/reference-designs/tkg-nsxt-airgap-ra.md:290]:
```suggestion
|TKO-ALB-SE-007|Set 'Placement across the Service Engines' setting to 'distributed'.|This allows for maximum fault tolerance and even utilization of capacity.| None|
```

File: src/reference-designs/tkg-nsxt-airgap-ra.md [hunk 4]
```diff
@@ -256,6 +256,40 @@ The following table provides a list of firewall rules based on the assumption th
...
+|TKO-ALB-SE-008|Set the SE size to a minimum 2vCPU and 4GB of Memory|This configuration should meet the most generic use case | For services that require higher throughput, these configuration needs to be investigated and modified accordingly.
```
Comment (reshmarajkumar-vmw) [inline src/reference-designs/tkg-nsxt-airgap-ra.md:291]:
```suggestion
|TKO-ALB-SE-008|Set the SE size to a minimum 2vCPU and 4GB of Memory.|This configuration should meet the most generic use case. | For services that require higher throughput, these configuration needs to be investigated and modified accordingly.|
```

File: src/reference-designs/tko-on-vsphere-with-tanzu.md [hunk 5]
```diff
@@ -281,14 +301,19 @@ The following table provides recommendations for configuring NSX Advanced Load B
...
+|TKO-Advanced Load Balancer-0010|Configure backup for the NSX ALB Controller cluster.|Backups are required if the NSX ALB Controller becomes inoperable or if the environment needs to be restored from a previous state.|To store backups, a SCP capable backup location is needed. SCP is the only supported protocol currently.|
+|TKO-Advanced Load Balancer-0011|Initial setup should be done only on one NSX ALB controller VM out of the three deployed to create an NSX ALB controller cluster.|NSX ALB controller cluster is created from an initialized NSX ALB controller which becomes the cluster leader.<br> Follower NSX ALB controller nodes need to be uninitialized to join the cluster.|NSX ALB controller cluster creation fails if more than one NSX ALB controller is initialized.|
+|TKO-Advanced Load Balancer-0012|Configure Remote logging for NSX ALB Controller to send events on Syslog.|For operations teams to be able to centrally monitor NSX ALB and escalate alerts events must be sent from the NSX ALB Controller|  Additional Operational Overhead.<br> Additional infrastructure Resource.
```
Comment (reshmarajkumar-vmw) [inline src/reference-designs/tko-on-vsphere-with-tanzu.md:315]:
```suggestion
|TKO-Advanced Load Balancer-0012|Configure Remote logging for NSX ALB Controller to send events on Syslog.|For operations teams to be able to centrally monitor NSX ALB and escalate alerts events must be sent from the NSX ALB Controller|  Additional Operational Overhead.<br> Additional infrastructure Resource.  |
```

File: src/reference-designs/tko-on-vsphere-with-tanzu.md [hunk 6]
```diff
@@ -281,14 +301,19 @@ The following table provides recommendations for configuring NSX Advanced Load B
...
+|TKO-Advanced Load Balancer-0011|Initial setup should be done only on one NSX ALB controller VM out of the three deployed to create an NSX ALB controller cluster.|NSX ALB controller cluster is created from an initialized NSX ALB controller which becomes the cluster leader.<br> Follower NSX ALB controller nodes need to be uninitialized to join the cluster.|NSX ALB controller cluster creation fails if more than one NSX ALB controller is initialized.|
+|TKO-Advanced Load Balancer-0012|Configure Remote logging for NSX ALB Controller to send events on Syslog.|For operations teams to be able to centrally monitor NSX ALB and escalate alerts events must be sent from the NSX ALB Controller|  Additional Operational Overhead.<br> Additional infrastructure Resource.  
+|TKO-Advanced Load Balancer-0013|Use LDAP/SAML based Authentication for NSX ALB| Helps to Maintain Role based Access Control | Additional Configuration is required
```
Comment (reshmarajkumar-vmw) [inline src/reference-designs/tko-on-vsphere-with-tanzu.md:316]:
```suggestion
|TKO-Advanced Load Balancer-0013|Use LDAP/SAML based Authentication for NSX ALB| Helps to Maintain Role based Access Control | Additional Configuration is required|
```

File: src/reference-designs/tko-on-vsphere-with-tanzu.md [hunk 7]
```diff
@@ -363,7 +389,7 @@ See [Sizing Service Engines](https://avinetworks.com/docs/20.1/sizing-service-en
 
 Multiple performance vectors or features may have an impact on performance.  For example, to achieve 1 Gb/s of SSL throughput and 2000 TPS of SSL with EC certificates, NSX Advanced Load Balancer recommends two cores.
 
-NSX Advanced Load Balancer Service Engines may be configured with as little as 1 vCPU core and 1 GB RAM, or up to 36 vCPU cores and 128 GB RAM.
+NSX Advanced Load Balancer Service Engines may be configured with as little as 1 vCPU core and 2 GB RAM, or up to 64 vCPU cores and 256 GB RAM. It is recommended for a Service Engine to have at least 4 GB of memory when Geo
```
Comment (reshmarajkumar-vmw) [inline src/reference-designs/tko-on-vsphere-with-tanzu.md:392]: <<Is this sentence incomplete?>>

**Review by reshmarajkumar-vmw — COMMENTED** [review]
Please find comments inline

File: src/reference-designs/tkg-nsxt-airgap-ra.md (hunk 3 above)
Comment (rahulk10-vmware) [reply src/reference-designs/tkg-nsxt-airgap-ra.md:290]: TKO-ALB-SE-007 decision is not related this RA.

File: src/reference-designs/tkg-nsxt-airgap-ra.md (hunk 4 above)
Comment (rahulk10-vmware) [reply src/reference-designs/tkg-nsxt-airgap-ra.md:291]: TKO-ALB-SE-008 decision is not related to this RA.

**Review by reshmarajkumar-vmw — APPROVED** [review]

**Comment by nithink-vmw** [comment]
Squash & Merge

**Pull request merged**